<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UITextFieldPropertyGridWidget</class>
 <widget class="QWidget" name="UITextFieldPropertyGridWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>225</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>225</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>299</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLineEdit" name="textLineEdit">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>26</y>
      <width>245</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="textLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Text:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="fontSizeSpinBox">
    <property name="geometry">
     <rect>
      <x>234</x>
      <y>60</y>
      <width>57</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="fontNameLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Font:</string>
    </property>
   </widget>
   <widget class="QColorButton" name="textColorPushButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>200</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="fontColorLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>101</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Font Color:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fontSelectButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>53</y>
      <width>181</width>
      <height>38</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {text-align:right; font-size: 11px; border-style: outset; border: 1px solid #969696; padding: 5px; border-radius: 4px; background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #e3e3e3, stop: 1 #f9f9f9);} 
QPushButton:pressed {background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #7e7e7e, stop: 1 #f9f9f9); }</string>
    </property>
    <property name="text">
     <string>default font</string>
    </property>
   </widget>
   <widget class="QColorButton" name="shadowColorButton">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>158</y>
      <width>185</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="offsetYLabel">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>130</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Y:</string>
    </property>
   </widget>
   <widget class="QLabel" name="offsetXLabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>X:</string>
    </property>
   </widget>
   <widget class="QLabel" name="shadowColorLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Shadow Color:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="shadowOffsetXSpinBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>126</y>
      <width>57</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-99</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="shadowOffsetYSpinBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>126</y>
      <width>57</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-99</number>
    </property>
   </widget>
   <widget class="QLabel" name="shadowOffsetLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Shadow offset:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="alignComboBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>184</y>
      <width>209</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="AlignLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Align:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QColorButton</class>
   <extends>QPushButton</extends>
   <header>qcolorbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>textLineEdit</tabstop>
  <tabstop>fontSelectButton</tabstop>
  <tabstop>fontSizeSpinBox</tabstop>
  <tabstop>textColorPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
